Makefile.am

```diff
@@ -664,7 +668,13 @@ EXTRA_DIST += \
 	.travis.yml \
 	.mailmap
 
-@INTLTOOL_POLICY_RULE@
+if HAVE_ITSTOOL
+%.policy: %.policy.in
+	$(AM_V_ITSTOOL)itstool -j $< $(wildcard $(top_srcdir)/po/*.gmo) -o $@
+else
+%.policy: %.policy.in
+	$(AM_V_ITSTOOL)echo "*** itstool support required to build target $@" && false
```

Comment:
Can we fallback to just using `cp` here?

I mean now that we use real `<description>` instead of a fake `<_description>`, I think the file would just work (without the internationalization support) if that is not available...

I'll take a closer look at the whole PR later (but, at a glance, looks good!)